src/Negotiation/AbstractNegotiator.php

```diff
@@ -109,7 +109,7 @@ public function getOrderedElements($header)
      * @param AcceptHeader $priority
      * @param integer      $index
      *
-     * @return Match|null Headers matched
+     * @return AcceptMatch|null Headers matched
```

Comment:
As it's already a BC break, maybe it's the good time to add return types (in another PR)?

Reply:
Why in another PR? As you said, this already is a BC break.

Reply:
To keep the PRs small and focused.